### PR TITLE
EVG-5984 Add support for Google analytics

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: service/templates/admin.html [hunk 1]
```diff
@@ -822,6 +823,25 @@ <h2 class="modal-title">[[modalTitle]]</h2>
...
 							</md-card>
+            </section>
+
+            <section layout="row" flex>
+							<md-card flex=50 id="google_analytics" style="height:280px">
+								<md-card-title>
+									<md-card-title-text>
+										<span>Google Analytics</span>
+									</md-card-title-text>
+									<md-button ng-click="clearSection('splunk')">
```
Comment (ybrill) [inline service/templates/admin.html]: does this button do what we want it to do?

Comment (bsamek) [reply service/templates/admin.html]: fixed

File: config.go [hunk 2]
```diff
@@ -61,6 +61,7 @@ type Settings struct {
 	Expansions         map[string]string         `yaml:"expansions" bson:"expansions" json:"expansions"`
 	ExpansionsNew      util.KeyValuePairSlice    `yaml:"expansions_new" bson:"expansions_new" json:"expansions_new"`
 	GithubPRCreatorOrg string                    `yaml:"github_pr_creator_org" bson:"github_pr_creator_org" json:"github_pr_creator_org"`
+	GoogleAnalytics    string                    `yaml:"google_analytics" bson:"google_analytics" json:"google_analytics"`
```
Comment (john-m-liu) [inline config.go]: Should this end in "ID?"

File: service/templates/admin.html [hunk 3]
```diff
@@ -822,6 +823,25 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</md-input-container>
 								</md-card-content>
 							</md-card>
+            </section>
+
+            <section layout="row" flex>
```
Comment (john-m-liu) [inline service/templates/admin.html]: Can this go next to the jira section in the right column? There is an unnecessary hole there right now

File: config.go (hunk 2 above)
Comment (bsamek) [reply config.go]: done

File: service/templates/admin.html (hunk 3 above)
Comment (bsamek) [reply service/templates/admin.html]: done

**Pull request merged**